### DISCUSSION TITLE
change Kurdish (Soranî) text to کوردی ناوڕاست

### DISCUSSION THREAD

**Discussion by iamyahia** [photopea/photopea#5887] [Ideas]
### hey photopea team

As a native speaker of Kurdish, I've been working extremely hard over the past three days to add Kurdish-Sorani to the new language for Photopea.
Nevertheless, I should note that several other languages added to the Photopea website's language area were added by their native characters, such as: (فارسی or العربیة ) , thus I suggest that to change (((  Kurdish (Soranî) to ** کوردی ناوڕاست ** ))  in [this section](https://www.photopea.com/translate/) .

I would much appreciate it if you changed this little text.

**Comment by photopea** [comment]
I have renamed it to "کوردیی ناوەندی" (according to the Wikipedia) - is it ok?

**Reply by iamyahia** [reply]
:fire:   :fire:  that's amazing :fire:  :fire:  
thank you so much :heart_decoration:

**Comment by photopea** [comment]
Thank you for your hard work with translating Photopea :)
